charts/gitlab-ci-pipelines-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 name: gitlab-ci-pipelines-exporter
-version: 0.2.12
+version: 0.3.0
```

Comment:
same as for #41!

Reply:
Done :)

Just a nit but this isn't correct according to [semver](https://semver.org/), this a backwards compatible feature so it should be a minor bump, not patch.

Reply:
thanks! yeah my bad I should probably mention on this repo as well that I tend to use 0ver for these projects: https://github.com/mvisonneau/gitlab-ci-pipelines-exporter/blob/main/CHANGELOG.md?plain=1#L5-L6